modules/data.atmosphere/R/download.NEONmet.R

```diff
@@ -0,0 +1,486 @@
+##' Download NEON Site Met CSV files
+##'
+##' @name download.NEONmet
+##' @title download.NEONmet
+##' @export
+##' @param site the NEON ID of the site to be downloaded, used as file name prefix. 
+##' The 4-letter SITE code  in \href{http://www.neonscience.org/science-design/field-sites/list}{list of NEON sites}
+##' @param outfolder location on disk where outputs will be stored
+##' @param start_date the start date of the data to be downloaded. Format is YYYY-MM-DD (will only use the year and month of the date)
+##' @param end_date the end date of the data to be downloaded. Format is YYYY-MM-DD (will only use the year and month part of the date)
+##' @param overwrite should existing files be overwritten
+##' @param verbose should the function be very verbose
+##' @importFrom PEcAn.utils fqdn logger.debug logger.error logger.warn logger.severe
+
+download.NEONmet <- function(sitename, outfolder, start_date, end_date, 
+                                  overwrite = FALSE, verbose = FALSE,  ...) {
+
+  if (!file.exists(outfolder)) {
+    dir.create(outfolder)
+  }
+  
+  #Check if site is a NEON site
+  site <- sub(".* \\((.*)\\)", "\\1", sitename)
+  siteinfo <- nneo::nneo_site(site)
+  if (!exists("siteinfo")) {
+    logger.error("Could not get information about", sitename, ".", "Is this a NEON site?")
```

Comment:
I thought the @importsfrom line in the top made it so that those functions did not need library declaration, no?